_pages/security/dynamic-scanning.md

```diff
@@ -5,29 +5,58 @@ parent: Security
 
+<img src="../../assets/images/zap_home.png" alt="zap home screen"/>
+
 ## Usage
 
-You will need a running application to test, which you will want to be as production-like as possible. Before testing, make sure to let your team know, as they may see a spike in requests and errors.
+You will need a running application to test, which you will want to be as production-like as possible. The 'Attack' scripts shouldn't do anything destructive, but will cause a spike in requests and errors. **Inform your team and #devops before running it on live (not locally-hosted) sites.**
 
 1. [Install the ZAP desktop app](https://github.com/zaproxy/zaproxy/wiki/Downloads).
 1. In the Quick Start tab, enter the URL of your application.
     * While you will need to run against production for the official ATO, you can also try out ZAP running against a local server.
-        * If you're using Rails on Mac, start it with
-
-            ```bash
-            rails server --binding 0.0.0.0
-            ```
 
 1. Click "Attack".
     * You should see a bunch of URLs start to pop up in the Spider tab, then the Active Scan tab.
     * This will take a while.
-    * There is a progress bar in the Spider and Active Scan tabs. You can click the <img class="inline" src="../../assets/zap_graph.png" alt="small graph icon"/> button in the latter to get more details.
+    * There is a progress bar in the Spider and Active Scan tabs. You can click the <img class="inline" src="../../assets/images/zap_graph.png" alt="small graph icon"/> button in the latter to get more details.
 1. Wait for the "Progress" area to say say "Attack complete".
 1. Open the Alerts tab.
 
 You can now click on each alert to see more details.
 
+### Examining the Results
+
+#### The Spider
+
+<img src="../../assets/images/spider_results.png" alt="spider results"/>
+As configured, the Quick Start Attack scan does not follow links to other domains or subdomains. If your project uses either, you will want to click <img class="inline" src="../../assets/images/zap_options.png" alt="small options icon"/> and update the options to include the domains & subdomains within the scope.
+
+#### Alerts
+
+<img src="../../assets/images/alert_results.png" alt="alert results"/>
+The Alerts pane lists all alerts discovered while scanning the site. The red and orange-flagged alerts must be taken care of before the application can be ATO'd. You have a little more flexibility when dealing with the yellow and blue flags, but all of them must be either corrected or, in the case of false positives, documented.
+
+Some of the alerts are caused by running the application with the debugging web server instead of the approved production server.
+
 ### Authentication
 
+Testing the application in an authenticated state is easiest when done with ZAP acting as a Man In The Middle Proxy. Setting up with Firefox via the "Plug-n-Hack" extension is available right from the Quick Start page.
```

Comment:
Okay, uninstalling and attempting to re-install didn't work. :/ I'll adjust the recommendation. Good catch.